### ISSUE TITLE
(Units) Micros can't be typed on calculator 

### ISSUE THREAD

**Issue by Moddingear** [numworks/epsilon#1803]
When typing a unit, such as micrometers, the calculator requires a mu, which can't be typed on the calculator and can only be had by going through the menus, and only with certain units. 
I suggest adding "u" as an alias to mu.

**Comment by MixedMatched** [comment]
Another option to fix this could be adding Mu on the bottom half of the keyboard in the shift layer.

**Comment by artaxxx** [comment]
Thanks @Moddingear for this issue. There's indeed something to do here. I don't think it deserved a hardware update but we could find a solution to input special characters

**Comment by A2drien** [comment]
A possible solution would be to add the sign µ with shift + 7, since alpha + 7 gives m.

In the same logic, the Ω used for the Ohm would be usable with shift + 9, since alpha + 9 gives o.

In order not to disturb users, one could imagine that these characters are added on a new revision of the calculator.

**Comment by RedGl0w** [comment]
> A possible solution would be to add the sign µ with shift + 7, since alpha + 7 gives m.
> 
> In the same logic, the Ω used for the Ohm would be usable with shift + 9, since alpha + 9 gives o.
> 
> In order not to disturb users, one could imagine that these characters are added on a new revision of the calculator.

Clearly not the best solution
If a new revision of the keyboard is necessary, a "char" button like on HP prime, which would open a menu with all the chars would be way more powerful

**Comment by VladPutinEu** [comment]
> > A possible solution would be to add the sign µ with shift + 7, since alpha + 7 gives m.
> > In the same logic, the Ω used for the Ohm would be usable with shift + 9, since alpha + 9 gives o.
> > In order not to disturb users, one could imagine that these characters are added on a new revision of the calculator.
> 
> Clearly not the best solution
> If a new revision of the keyboard is necessary, a "char" button like on HP prime, which would open a menu with all the chars would be way more powerful

I agree with you. This should be added in the next version of the calculator. But for now, maybe a software fix would do.

**Comment by adrienferreira** [comment]
If anybody wants to do what @viveksh1 suggests, the file to change is currently located in `ion/src/shared/layout_events.cpp`.
Just replace `U()` by `T("Ω")` where the `shift + 9` line is located, same for `T("μ")` where `shift + 7` is.
It's literally two lines to change, I can share my diff if anyone's interested.
